packages/core/src/forms/control/control.element.scss

```diff
@@ -24,6 +24,10 @@ $fallback-width: calc(#{$cds-global-layout-space-xxl} * 2);
   --max-width: #{$fallback-width};
 }
 
+:host([layout*='vertical']) cds-internal-control-label {
+  --label-width: 100%;
+}
+
```

Comment:
I wasnt sure if there was really a good test for this since its purely styles and no behavior. Probably something to be covered my our screenshot testing soon.